README.md

```diff
@@ -48,21 +48,6 @@ Once they get modified it is necessary recompile them. To do that we can use `np
 Both the [CircleCI build](.circleci/config.yml) and the [Docker files](docker-compose.yml) leverage a [Makefile](Makefile), facilitating the execution of more complex routines.
 You can have a look at the [Makefile](Makefile) to know more about the exact commands used to build the website.
 
-## Environment variables
```

Comment:
Since you are removing this whole block, please add somewhere on line 17 that extended information regarding the ENVs are inserted in the .env.example file so that a user knows it. Thanks